src/presets.js

```diff
@@ -39,55 +39,55 @@ const octokit = new Octokit({
 
 // prettier-ignore
```

Comment:
It looks like you've reformatted this in a way that ignores this `prettier-ignore` - the reason for the custom formatting is that the strings are in essence code which is nicely indented for readability and the auto-formatting has clobbered that readability now

Reply:
Sure, I can change that. I just ran `npm run prettify` which does not seem to respect the ignore...